eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -265,12 +265,18 @@ private List<Path> transpile(
      * {@link java.nio.file.AccessDeniedException}, which could crash the build.
      * _____
      * @param java The list of java files.
-     * @todo #2375:90min. Add concurrency tests for the TranspileMojo.cleanUpClasses method.
-     *  We should be sure that the method works correctly in a concurrent environment.
-     *  In order to do so we should add a test that will run the cleanUpClasses method in
-     *  multiple threads and check that the method works correctly without exceptions.
-     *  We can apply the same approach as mentioned in that post:
-     *  <a href="https://www.yegor256.com/2018/03/27/how-to-test-thread-safety.html">Post</a>
+     * @todo #2375:90min. Implement a mechanism for "internal" and "external" classes.
```

Comment:
@l3r8yJ, could you please provide more context? Imagine someone opening this issue in the future; they should be able to understand what to do at first glance:

1. I still don't understand why we might need "internal" and "external" classes, or why we need to implement them.
2. Why should "internal" classes be included in the catalog?
3. Which specific catalog are you referring to?
4. Why should "external" classes be placed in a different directory?
5. Are "catalog" and "directory" the same entities in this context?